mandatory/3-playing-computer.js

```diff
@@ -8,11 +8,11 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  3. What is printed to the console? 2,6,4,9,6,18,8
+  4. How many times is "f1" called? 2
+  5. How many times is "f2" called? 2
```

Comment:
There are 5 iterations of the for loop and the index which starts at 0 is incremented by one with each loop until it reaches 5, then it stops. For each iteration a check is done on the current index. if it is even the f2 function is called. The index is 0,2 and 4 at various iterations of the loop and so the f2 function is called 3 times. The index is also 1 and 3 at certain iterations and so the f1 function is called twice.